IaC/PnP/monitoring/aks/online/aks_secure_baseline/cluster-baseline-settings/ns-a0008.yaml

```diff
@@ -0,0 +1,4 @@
+kind: Namespace
```

Comment:
Similar note to above that the namespace for one specific application is in the cluster-baseline-settings. I'm assuming they did this so that the NetworkPolicy will be able to deploy, but we should see if they can be moved